mono/metadata/threadpool-worker-default.c

```diff
@@ -356,6 +356,25 @@ mono_threadpool_worker_request (void)
 	mono_refcount_dec (&worker);
 }
 
+#if ENABLE_NETCORE
+gint64 mono_threadpool_worker_get_completed_threads_count (void)
```

Comment:
should be called `mono_threadpool_worker_get_completed_work_items_count`, I think

Reply:
Can this be faster somehow? Provide an address that managed holds on to and returns repeatedly? An icall to read a global variable seems, slow. Perhaps there can be some data driven relationship with method-to-ir to inline these, without being a special case? Similar to the slow increment I introduced as part of my first or second mono commit, that I still regret, perf-wise.

Reply:
ps: Get it to master independent of my question, of course, but it might be nice to have a better system sooner than later, for binding to and incrementing and atomically incrementing global int32, int64, gsize, something like that.